### PR TITLE
Change Webpack production configuration so that *.map.js files will be generated.

### PR DESCRIPTION
**Pull request by tischsoic**
Changed Webpack production configuration so that *.map.js files will be generated.

### CODE REVIEW

**Review by sunpietro — APPROVED** [review]

**Comment by sunpietro** [comment]
Please, resolve conflicts. Probably branch has to be rebased

**Comment by lserwatka** [comment]
@sunpietro can we merge it?

File: webpack.dev.js [hunk 1]
```diff
@@ -2,5 +2,4 @@ const common = require('./webpack.common');
 const merge = require('webpack-merge');
 
 module.exports = merge(common, {
```
Comment (dew326) [inline webpack.dev.js:4]: Merging with an empty object?

Comment (tischsoic) [reply webpack.dev.js:4]: Yes, configuration for `dev` now equals the one from `common`, but with this empty object you have everything set up to change that state.

E.g. in the near future introduction of tests may need different config. Etc.

**Review by dew326 — APPROVED** [review]

**Pull request merged**